model-archiver/model_archiver/tests/unit_tests/test_model_packaging_utils.py

```diff
@@ -138,27 +136,17 @@ def __init__(self, **kwargs):
 
         author = 'ABC'
         email = 'ABC@XYZ.com'
-        engine = EngineType.MXNET.value
         model_name = 'my-model'
         handler = 'a.py::my-awesome-func'
         serialized_file = 'model.pt'
         model_file = 'model.pt'
         version = "1.0"
         source_vocab = None
 
-        args = Namespace(author=author, email=email, engine=engine, model_name=model_name, handler=handler,
+        args = Namespace(author=author, email=email, model_name=model_name, handler=handler,
```

Comment:
@harshbafna How can user's specify the author/email? The CLI doesn't have any documented steps for this: https://github.com/pytorch/serve/blob/master/model-archiver/README.md#arguments

Reply:
Cleaned up unused params.